docs/roadmap.md

```diff
@@ -4,8 +4,9 @@ Caveat: contains forward-looking statements that may or may not match reality. [
 
 ## Release 1.2
 
-- Support for AWS/kops platform ([#292](https://github.com/bitnami/kube-prod-runtime/issues/292))
```

Comment:
We should bump this to 1.3, rather than drop it entirely. (imo - assuming that's our plan)

Reply:
Good point. Added back to 1.3